wcivf/apps/elections/models.py

```diff
@@ -145,16 +145,20 @@ class Post(models.Model):
 
     objects = PostManager()
 
-    def friendly_name(self):
-        if self.ynr_id == "ref.2016-06-23":
-            return self.label
-        return "{} for {}".format(self.election.for_post_role, self.area_name)
-
 class PostElection(models.Model):
     post = models.ForeignKey(Post)
     election = models.ForeignKey(Election)
     contested = models.BooleanField(default=True)
 
+
+    def friendly_name(self):
+        # TODO clearly not every election is a by-election
+        # This will get better once we import more info from YNR/EE
+        return "{} ward by-election".format(
```

Comment:
Given we are going to have to deal with a Welsh Assembly by-election soon (before Feb 7th - source: https://www.itv.com/news/wales/update/2017-11-20/alyn-deeside-vacant-formal-declaration/ ), lets not hard-code "ward" here. If we can pick up the post/geography type from the `Post` model, can we put in a special case? If not, lets stick to `return "{} by-election".format(` here.